packages/rancher-logging/generated-changes/overlay/templates/loggings/rke2/configmap.yaml

```diff
@@ -10,7 +10,8 @@ data:
     [INPUT]
         Name              systemd
         Tag               rke2
-        Systemd_Filter    _SYSTEMD_UNIT=rke2.service
+        Path              /var/log/journal
+        Systemd_Filter    _SYSTEMD_UNIT=rke2-server.service
```

Comment:
Are all types of rke2 server ? Would agent/node have this name too.

Reply:
all the systemd logs would fall under that unit.  I did not see any others mentioned in the rke2 docs.

Reply:
The rest are under containers

Reply:
+1 to @paynejacob they use the same systemd unit